### PR TITLE
Update pom.xml

### PR DESCRIPTION
**Pull request by RAGHUNANDANJP**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>3.4.1</version>
+                <version>3.4.0</version>
```
Comment (jlerbsc) [inline pom.xml:40]: Why downgrade the plugin version?

**Comment by RAGHUNANDANJP** [comment]
Hi,

I was learning about it & when I run the package i got Maven Version is not
in the allowed range then I have changed the version.

**Comment by jlerbsc** [comment]
I just clone the repository and run mvn package with any error.

> [INFO] --- maven-shade-plugin:3.4.1:shade (default) @ javaparser-maven-sample ---
[INFO] Including com.github.javaparser:javaparser-core:jar:3.25.4 in the shaded jar.
[INFO] Dependency-reduced POM written at: javaparser-maven-sample\dependency-reduced-pom.xml
[WARNING] javaparser-core-3.25.4.jar, javaparser-maven-sample-1.0-SNAPSHOT.jar define 1 overlapping resource:
[WARNING]   - META-INF/MANIFEST.MF
[WARNING] maven-shade-plugin has detected that some class files are
[WARNING] present in two or more JARs. When this happens, only one
[WARNING] single version of the class is copied to the uber jar.
[WARNING] Usually this is not harmful and you can skip these warnings,
[WARNING] otherwise try to manually exclude artifacts based on
[WARNING] mvn dependency:tree -Ddetail=true and the above output.
[WARNING] See https://maven.apache.org/plugins/maven-shade-plugin/
[INFO] Attaching shaded artifact.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.101 s
[INFO] Finished at: 2023-08-26T09:12:00+02:00
[INFO] ------------------------------------------------------------------------

**Pull request closed without merging**